### ISSUE TITLE
sftp.json can also be used via Settings Sync

### ISSUE THREAD

**Issue by csaeum** [liximomo/vscode-sftp#844]
Hello, I think your extension is really good and easy to use.

I am currently using Atom and would switch to VSCode.

With Atom, the GIT Sync also synchronizes the SFTP or FTP data. Unfortunately that doesn't seem to work with your extension at the moment.

Did I miss something or can you tell me what to do to get this?

**Comment by ghost** [comment]
I don't think this plugin is being maintained anymore.  It hasn't been committed to since November 2019.
